build_tools/scripts/update_submodules.sh

```diff
@@ -37,8 +37,29 @@ if [ -n "$git_status" ]; then
   exit 1
 fi
 
-echo "Pulling latest from remote"
-git pull
+master_url="$(git remote get-url master)"
+upstream_url="https://github.com/google/iree.git"
```

Comment:
Let's try to follow the Google shell style guide (https://google.github.io/styleguide/shell.xml). I think this would be a constant, so how about all caps

Reply:
Thanks.

Reply:
Changed it now. Sorry, got side-tracked by something else.

Reply:
No worries!